apps/financial-aid/web-veita/src/components/Profile/MunicipalityProfile.tsx

```diff
@@ -0,0 +1,256 @@
+import React from 'react'
+import { Box, Button, Link, Text } from '@island.is/island-ui/core'
+
+import * as styles from './Profile.css'
+import * as headerStyles from '@island.is/financial-aid-web/veita/src/components/ApplicationHeader/ApplicationHeader.css'
+import * as tableStyles from '../../sharedStyles/Table.css'
+import cn from 'classnames'
+
+import {
+  AidType,
+  Municipality,
+  Staff,
+} from '@island.is/financial-aid/shared/lib'
+
+import {
+  TableHeaders,
+  TableBody,
+  TextTableItem,
+  ActivationButtonTableItem,
+} from '@island.is/financial-aid-web/veita/src/components'
+
+interface MunicipalityProfileProps {
+  municipality: Municipality
+}
+
+const MunicipalityProfile = ({ municipality }: MunicipalityProfileProps) => {
+  const aidTableBody = (value: AidType) => {
+    switch (value) {
+      case AidType.OWNPLACE:
```

Comment:
This is very nice, maybe have two const here, like 
const headline = 'h5'
const smallText = 'small' ,
so if we need to change we only need do it there? 
I dont know just a suggestion i thought of 🤷‍♀️

Reply:
Makes sense, will do :)